pkgs/development/compilers/graalvm/community-edition/mkGraal.nix

```diff
@@ -194,9 +164,15 @@ let
         # copy-paste openjdk's preFixup
         # Set JAVA_HOME automatically.
         mkdir -p $out/nix-support
-        cat <<EOF > $out/nix-support/setup-hook
+        cat > $out/nix-support/setup-hook << EOF
           if [ -z "\''${JAVA_HOME-}" ]; then export JAVA_HOME=$out; fi
         EOF
+        ${
+        lib.optionalString (stdenv.isLinux) ''
+          ${copyClibrariesToOut "$out/lib/svm/clibraries"}
+          ${copyClibrariesToLib}
```

Comment:
I don't think those two functions make much sense anymore. This was when we used to apply a specific configuration for GraalVM 8 vs GraalVM 11.

So I think we can just add the text directly here. WDYT?

Reply:
Jep.